CHANGES.rst

```diff
@@ -8,6 +8,10 @@
 Changes
 =======
 
+Version 1.0.1 (released 2022-11-14)
+
+- allow kwargs in VocabularyCF
```

Comment:
very minor:

```suggestion
- Allow kwargs in VocabularyCF
```

Should also change L17. To be consistent with the rest of the file